packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -3,7 +3,6 @@ syntax = "proto3";
 option optimize_for = CODE_SIZE;
 
 import "google/protobuf/empty.proto";
-import "google/protobuf/any.proto";
```

Comment:
Good improvement 👍 Maybe worth mentioning in the PR description as this isn't strictly related to this PR.